packages/storage-ui/src/Components/Modules/AddCIDModal.tsx

```diff
@@ -147,14 +155,31 @@ const AddCIDModal = ({ modalOpen = false, close }: IAddCIDModuleProps) => {
           sm={12}
           className={classes.input}
         >
+          <Typography
+            className={classes.heading}
+            variant="h5"
+            component="h5"
+          >
+            <Trans>Pin a CID</Trans>
+          </Typography>
+          <TextInput
+            label={t`Name`}
```

Comment:
I would love to explicitly say that the name is optional. 
```suggestion
            label={t`Name (optional)`}
```

Reply:
It's not an awesome feature to accept empty names and I find it somewhat wordy, but if you think it's better, I can live with that.